homing_missiles.py

```diff
@@ -109,12 +109,22 @@ def draw_entities(self, screen):
         pygame.draw.rect(screen, self.main_character_color_, main_character_rect)
         self.draw_friendly_projectiles(screen)
 
+    def determine_enemy_color(self, enemy):
+        enemy_type = enemy.get_type()
+        if enemy_type == "BasicGrunt":
+            return YELLOW
+        elif enemy_type == "NotSoBasicGrunt":
+            return WHITE
+
```

Comment:
I was debating that myself, and I decided that if anyone was to add a new `elif` they'd have to move that between the `else` and the previous `elif`. This solution lets the person just add a new `elif` to the end without moving anything else